requirements-dev.txt

```diff
@@ -2,10 +2,16 @@
 flake8
 mypy
 build
+black
 
 pytest
 pytest-asyncio
 pytest-cov
-coverage-lcov
```

Comment:
Lcov seems to have some compatibility issues. If first complains that toml is not installed. After that it complains with that message: ImportError: cannot import name 'FnmatchMatcher' from 'coverage.files'

Reply:
fixed with [KIT-3375 Fix lcov compatibility issue](https://github.com/SamhammerAG/sag_py_auth_brand/pull/7/commits/59a1a5391c77d0fa1222b7a2fe790ca807a55652)